app/models/album.rb

```diff
@@ -1,20 +1,19 @@
 # encoding: UTF-8
 class Album < ActiveRecord::Base
-  has_many :images, dependent: :destroy
+  has_many :images, -> { order(:filename) }, dependent: :destroy
```

Comment:
@jforberg 
As far as I could see this should be better than the default scopes :)